### PR TITLE
frac fix and that things

### PR DESCRIPTION
**Pull request by latot**
Hi again, more functions.

Fixes https://github.com/cbm755/octsympy/issues/217

### CODE REVIEW

File: inst/@sym/fix.m [hunk 1]
```diff
@@ -0,0 +1,59 @@
+%% Copyright (C) 2015, 2016 Colin B. Macdonald and Lagu
```
Comment (cbm755) [inline inst/@sym/fix.m]: just your name

File: inst/@sym/fix.m [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+%% Example:
+%% @example
+%% @group
+%% y = fix(sym(3)/2)
+%%   @result{} y = (sym) 1
+%% @end group
+%% @end example
+%%
+%% Note: this file is autogenerated: if you want to edit it, you might
+%% want to make changes to 'generate_functions.py' instead.
```
Comment (cbm755) [inline inst/@sym/fix.m]: remove this, this is not autogenerated.

File: inst/@sym/fix.m [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+%%
+%% @end deftypefn
```
Comment (cbm755) [inline inst/@sym/fix.m]: Can you add a `@seealso` block?

File: inst/@sym/fix.m [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+
+%!shared x, d
+%! d = 3/2;
+%! x = sym('3/2');
+
+%!test
+%! f1 = fix(x);
+%! f2 = fix(d);
+%! assert (isequal (f1, f2))
```
Comment (cbm755) [inline inst/@sym/fix.m]: delete the `!shared` bit and move d and x into your first test.

File: inst/@sym/frac.m [hunk 5]
```diff
@@ -0,0 +1,47 @@
+%% Copyright (C) 2016, 2016 Colin B. Macdonald and Lagu
```
Comment (cbm755) [inline inst/@sym/frac.m]: as above

File: inst/@sym/frac.m [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @deftypefn  {Function File} {@var{y} =} fix (@var{x})
+%% Symbolic fix function.
```
Comment (cbm755) [inline inst/@sym/frac.m]: change to `Return the fractional part of a symbolic expression.`

File: inst/@sym/frac.m [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+%% Example:
+%% @example
+%% @group
+%% y = frac(sym(3)/2)
+%%   @result{} y = (sym) 1/2
+%% @end group
+%% @end example
+%%
+%% Note: this file is autogenerated: if you want to edit it, you might
+%% want to make changes to 'generate_functions.py' instead.
```
Comment (cbm755) [inline inst/@sym/frac.m]: delete this, add seealso

File: inst/@sym/frac.m [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+%%
+%% @end deftypefn
+
+%% Author: Colin B. Macdonald
+%% Keywords: symbolic
```
Comment (cbm755) [inline inst/@sym/frac.m]: Either put your name or delete these two lines.  I'm leaning towards delete them.  I think I will stop adding these to new functions as well.

File: inst/@sym/frac.m [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+function y = frac(x)
+  y = uniop_helper (x, 'frac');
+end
+
+%!test
+%! f1 = frac(sym(11)/10);
+%! f2 = sym(1)/10;
+%! assert (isequal (f1, f2))
```
Comment (cbm755) [inline inst/@sym/frac.m]: Can we add a test for negative numbers?  Maybe the docs should also mention what it does for negative numbers (IIRC, there are mulitple "standards" for this function)

File: inst/@sym/round.m [hunk 10]
```diff
@@ -0,0 +1,59 @@
+%% Copyright (C) 2016, 2016 Colin B. Macdonald and Lagu
```
Comment (cbm755) [inline inst/@sym/round.m]: as above

File: inst/@sym/round.m [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+%% Example:
+%% @example
+%% @group
+%% y = round(sym(-27)/10)
+%%   @result{} y = -3
+%% @end group
+%% @end example
+%%
+%% Note: this file is autogenerated: if you want to edit it, you might
+%% want to make changes to 'generate_functions.py' instead.
```
Comment (cbm755) [inline inst/@sym/round.m]: delete, add `@seealso`.

File: inst/@sym/round.m [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+
+%% Author: Colin B. Macdonald
+%% Keywords: symbolic
+
+function y = round(x)
+  y = uniop_helper (x, 'round');
+end
+
+
+%!shared x, d
```
Comment (cbm755) [inline inst/@sym/round.m]: as above, remove this

**Comment by cbm755** [comment]
This looks pretty good!  I made some comments throughout...

Looks some of the new tests are failing.

**Comment by latot** [comment]
okidoki.

**Comment by cbm755** [comment]
This is great!  I think the tests fail because of old SymPy.  I sent you a PR: if you merge, I think this is probably ready to go.

**Comment by cbm755** [comment]
Sweet, merging.
